index.html

```diff
@@ -62,7 +118,7 @@ <h2>Ссылки на соцсети</h2><!--скрыть ч/з visually hidden
       <p>
         Website by
         <a href="https://htmlacademy.ru/intensive/htmlcss" target="_blank">
-          <img src="abc.svg" width="100" height="100" alt="Логотип htmlacademy">
+          <img  src="abc.svg" width="100" height="100" alt="Логотип htmlacademy">
```

Comment:
```suggestion
          <img src="abc.svg" width="100" height="100" alt="Логотип htmlacademy">
```

Было же правильно до этого. Зачем лишний пробел перед атрибутом `src`?